### PR TITLE
Xenobio update

### PR DESCRIPTION
**Pull request by LongSleeves**
Renames some xenobio potions for easy reading in the code
Buffs the amount of honey, uranium and diamonds gotten thru xenobio.
Repaths various xenobio goods on backend
Adds a still work in progress slime dye vat
Adds in a few slime potions that grant buffs to clothes
Thanks to Hound who gave a few sprites to work with for the new bottles and dye vat

### CODE REVIEW

File: code/modules/reagents/recipes/recipes_slime.dm [hunk 1]
```diff
@@ -146,15 +146,11 @@
 	var/obj/item/stack/material/gold/G = new /obj/item/stack/material/gold
 	G.amount = 15
 	G.loc = get_turf(holder.my_atom)
-	var/obj/item/stack/material/platinum/P = new /obj/item/stack/material/platinum
-	P.amount = 5
-	P.loc = get_turf(holder.my_atom)
```
Comment (r4d6) [inline code/modules/reagents/recipes/recipes_slime.dm:151]: Why remove platinum from the reaction?

Comment (LongSleeves) [reply code/modules/reagents/recipes/recipes_slime.dm:151]: Sieg said its a lategame gateway material. Ideally I was going to move it to a different slime. For now I'm thinking of moving it to pink and pushing pinks potions to light pink so you don't feel like that entire line is just slime friend.

File: code/game/machinery/xenobio.dm [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+	user.visible_message(SPAN_DANGER("[user] starts to put [victim] into the slime compresser!"))
+	src.add_fingerprint(user)
+	if(do_after(user, 30, src) && victim.Adjacent(src) && user.Adjacent(src) && victim.Adjacent(user))
+		user.visible_message(SPAN_DANGER("\The [user] stuffs \the [victim] into the compresser!"))
+		flick("compressering", src)
+		new victim.coretype(user.loc)
+		qdel(victim)
+
+*/Dye vat and its various procs and checks --- Currently a work in progress but dosn't break anything ---
```
Comment (r4d6) [inline code/game/machinery/xenobio.dm:55]:
```suggestion
//Dye vat and its various procs and checks --- Currently a work in progress but dosn't break anything ---
```

Comment (LongSleeves) [reply code/game/machinery/xenobio.dm]: yea saw it. reversed it XD

**Pull request merged**